api/document/js/main.ts

```diff
@@ -13,23 +14,15 @@ import fetchDoc from './fetch-doc';
 // do with non-standard kinds of imports like CSS, so using require()
 // likely bypasses TypeScript and goes straight to webpack, which
 // deals with it correctly. I could be wrong, though. -AV
-declare function require(path: string): null;
```

Comment:
If we don't need to declare this anymore, how much of the comment above it should go, too?

Reply:
Oh, I figured the comment above still applied, because the fact that we're still using `require()` over `import` is still potentially confusing IMO... But as you mentioned above, since we're going to switch to just putting the CSS in our funky SCSS build pipeline thing, we can get rid of it very soon regardless!

Reply:
Uhh wait, now CircleCI is complaining about `require` being undefined.  I have no idea why.  Guess I'll re-introduce it for now...